Televi/Data/Remote/TeleviRDS.swift

```diff
@@ -33,47 +34,35 @@ extension TeleviAPI: TargetType {
     }
     
     var task: Moya.Task {
-            return .requestPlain
-        }
+        return .requestPlain
```

Comment:
Whenever you have only one line on a computed property you can omit the return keyword